src/components/MoneyRequestConfirmationList.js

```diff
@@ -103,6 +113,9 @@ function MoneyRequestConfirmationList(props) {
     // Prop functions pass props itself as a "this" value to the function which means they change every time props change.
     const {translate, onSendMoney, onConfirm, onSelectParticipant} = props;
 
+    // A flag and a toggler for showing the rest of the form fields
+    const [showAllFields, toggleShowAllFields] = useReducer((state) => !state, false);
```

Comment:
Why not `useState` here?

Reply:
I like this approach as it exposes a bool flag and a toggle function in one line. With `useState` you have declare a separate function to toggle the flag. Nice and clean.